src/tutorials/konnector/2fa.md

```diff
@@ -0,0 +1,62 @@
+You have successfully developped your connector but you realize that some accounts need two factor
+authentication (2FA) to work.
+
+Here is how you can add this to your connector
+
+## 2FA detection
+
+Every target website being different, you will have to develop it yourself. Some websites return a
+specific url, some others display a special html content.
+
+## Home Automatic Successull Login deactivation
+
+By default, the home will suppose the connector's login is a success after 8s, which is bad when
```

Comment:
```suggestion
By default, the "Home" application will suppose the connector's login is a success after 8s, which is bad when
```

Reply:
Home app or Harvest? I saw that we're using `Harvest` in the sequence diagram

Reply:
I have a dilemma here, Harvest actually means nothing for a connector developper I think